.github/workflows/ci.yml

```diff
@@ -25,6 +25,12 @@ jobs:
         with:
           fetch-depth: 0
 
+      - uses: actions/cache@v3 
+        id: cache-irfs
+        with:
+          path: ./pyirf/recources/irfs
+          key: ${{ runner.OS }}-irf-cache-${{ hashFiles('*.fit.gz') }}
```

Comment:
Does `hasFiles('*.fit.gz')` work with files in subfolders or does it need to be something like `'**/*.fits.gz'`?

Reply:
Quoting from this [stackoverflow](https://stackoverflow.com/questions/55110729/how-do-i-cache-steps-in-github-actions#57958803) on the topic:

> It is useful to prefix keys and restore keys with the OS and name of the cache, as it shouldn't load files for a different type  of cache or OS.

which is probably true but since we are only running a CI on ubuntu-latest this is somewhat unneeded. Should only trigger when ubuntu-latest changes what it refers to.

Not sure about the wildcards though, I test it. As of know it does not work but thats probably because of the `fit.gz` typo.

Reply:
> It is useful to prefix keys and restore keys with the OS and name of the cache, as it shouldn't load files for a different type of cache or OS.

That's because people usually cache build-artifacts, which are os specific. We have data files that are os independent

Reply:
Removed the os prefix, seems to work fine with `**/*.fits.gz`